src/lib/core/say.c

```diff
@@ -1259,7 +1259,7 @@ log_vsay(struct log *log, int level, const char *filename, int line,
 		log->on_log(level, filename, line, error, format, ap_copy);
 		va_end(ap_copy);
 	}
-	if (level > log->level) {
+	if (!say_log_level_is_enabled(level)) {
```

Comment:
Before this patch, we checked the level of the `log` object while now we check the global log level. This looks incorrect, because `log_vsay` deals with a `log` object. Why do you want to change this?